README.md

```diff
@@ -171,7 +171,40 @@
    Note: Please install the openssl with command `nmake install` before build libspdm.
    cmake -G"Visual Studio 16 2019" -DARCH=x64 -DTOOLCHAIN=VS2019 -DTARGET=Release -DCRYPTO=openssl -DENABLE_BINARY_BUILD=1 -DCOMPILED_LIBCRYPTO_PATH=<OPENSSL_PATH>/libcrypto.lib -DCOMPILED_LIBSSL_PATH=<OPENSSL_PATH>/libssl.lib ..
    ```
-
+### Armclang build on Windows.
+1) Install [MSYS2](https://www.msys2.org/).
+2) Install amrclang (change the default installation path from C:\Program Files\Arm\Development Studio 2022.1   to C:\ArmStudio to avoid bank space)
+3) Launch MSYS2 -> MSYS2 MINGW64.
+4) Install cmake and make.
+```
+pacman -S mingw-w64-x86_64-cmake
+pacman -S make
+```
+5) Setup build environment command
+```
+cd libspdm
+mkdir build
+cd build
+export PATH=$PATH:/c/ArmStudio/sw/ARMCompiler6.18/bin
+export CC=/c/ArmStudio/sw/ARMCompiler6.18/bin/armclang.exe
+export ARM_PRODUCT_DEF=/c/ArmStudio/sw/mappings/gold.elmap
+export ARMLMD_LICENSE_FILE=XXXX
+```
+6) Build arm command
+```
+cmake -G"MSYS Makefiles" -DARCH=arm -DTOOLCHAIN=ARM_CLANG -DTARGET=Debug -DCRYPTO=mbedtls ..

```

Comment:
I got error "CMake Error: CMAKE_AR was not found"

Reply:
Also error:
```
At least one of the variables CMAKE_SYSTEM_PROCESSOR or CMAKE_SYSTEM_ARCH
  must be set for ARMClang
```

Reply:
I have updated the README as temporary solution for this error:
`Add set(CMAKE_SYSTEM_ARCH "armv8-a") on the top of C:\msys64\mingw64\share\cmake\Modules\Compiler\ARMClang.cmake. The CMAKE_SYSTEM_ARCH is the target arch.`

If there is a better solution, I will update it.

Reply:
still failed

```
    /C/ArmStudio/sw/ARMCompiler6.18/bin/armlink.exe "CMakeFiles/cmTC_dcde2.dir/testCCompiler.o"  -o cmTC_dcde2.exe  --list=cmTC_dcde2.map
    Error: L6415E: Could not find a unique set of libraries compatible with this image.  Suggest using the --cpu option to select a specific library.
```